### PR TITLE
OSDOCS-3779 adding instructions to remove unhealthy etcd member

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.6+

Issue: [OSDOCS-3779](https://issues.redhat.com/browse/OSDOCS-3779)

Link to docs preview: http://file.rdu.redhat.com/lahinson/OSDOCS-3779-unhealthy-etcd/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member

Additional information: Adding two substeps and an IMPORTANT note.

### CODE REVIEW

**Review by tjungblu — APPROVED** [review]
love it, thanks for the quick turnaround @lahinson 

/lgtm

File: modules/restore-replace-stopped-etcd-member.adoc [hunk 1]
```diff
@@ -71,9 +71,43 @@ sh-4.2# etcdctl member list -w table
...
+| 757b6793e2408b6c | started |  ip-10-0-164-97.ec2.internal |  https://10.0.164.97:2380 |  https://10.0.164.97:2379 |
+| ca8c2990a0aa29d1 | started | ip-10-0-154-204.ec2.internal | https://10.0.154.204:2380 | https://10.0.154.204:2379 |
++------------------+---------+------------------------------+---------------------------+---------------------------+
+----
 +
 You can now exit the node shell.
++
+[IMPORTANT]
+====
+After you remove the member, the remaining etcd instances are rebooted, so the cluster might be unreachable for a short period.
```
Comment (GroceryBoyJr) [inline modules/restore-replace-stopped-etcd-member.adoc:109]: For your consideration: remove the "," in front of "so".

**Review by GroceryBoyJr — COMMENTED** [review]
/LGTM - only one tiny nit please.

**Review by EricPonvelle — COMMENTED** [review]
I had two small comments about the passive voice, but that's for your consideration only. lgtm

**Comment by lahinson** [comment]
@GroceryBoyJr and @EricPonvelle Thanks for the review! I made the suggested changes, which should now be visible from the preview link 👆

**Comment by lahinson** [comment]
Hi @mburke5678, thanks for merging this PR! I'm still learning about how this all works, and I have a question: when you cherry pick to enterprise-4.6, is the change applied to version 4.6 and every version after it? I ask because this change applies to version 4.6+ (i.e., 4.6, 4.7., 4.8, 4.9, and 4.10).

**Comment by mburke5678** [comment]
@lahinson No. Sorry I lost track of this one. We need to cherry pick to each version. Personally, I usually cherry pick to one branch to make sure that there are no errors.  Then cherry pick to the rest. Not sure why this cherry pick didn't work.

**Pull request merged**